packages/desktop-gui/src/runs/runs-list-empty.jsx

```diff
@@ -47,7 +47,7 @@ class RunsListEmpty extends Component {
       url: 'https://on.cypress.io/ci',
       params: {
         utm_medium,
-        utm_campaign: 'Run Guide',
+        utm_campaign: 'CI',
```

Comment:
Was this file changed intentionally?

Reply:
@JessicaSachs recommended that I pull latest `develop` into this branch since there were certain tests failing on master. These changes are from https://github.com/cypress-io/cypress/pull/17918/commits/fa4b9d7a72b7b9a7f51807b9db94cd554f4aa98f which fixes some of those tests.

Reply:
Talked with @JessicaSachs and reverted the merge from develop!